File: app/src/main/java/com/example/kalkulatorbmi/DeathActivity.java
package com.example.kalkulatorbmi;
import androidx.appcompat.app.AlertDialog;
import androidx.appcompat.app.AppCompatActivity;

import android.content.DialogInterface;
import android.content.Intent;
import android.os.Bundle;
import android.view.Display;
import android.view.View;
import android.webkit.WebView;
import android.widget.TextView;
import android.widget.EditText;
import android.widget.Button;

import com.anychart.AnyChart;
import com.anychart.AnyChartView;
import com.anychart.chart.common.dataentry.DataEntry;
import com.anychart.chart.common.dataentry.ValueDataEntry;
import com.anychart.charts.Pie;

import java.util.ArrayList;
import java.util.List;
import java.util.Random;

public class DeathActivity extends AppCompatActivity {

    AnyChartView anyChartView;

    String[] months = {"Jan","Feb","Mar"};
    int[] earnings = {12000, 8009, 10000};

    @Override
    protected void onCreate(Bundle savedInstanceState) {

                super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_death);

        anyChartView = findViewById(R.id.any_chart_view);

        setupPieChart();
    }

    public void setupPieChart() {
        Pie pie = AnyChart.pie();
        List<DataEntry> dataEntries = new ArrayList<>();

        for (int i = 0; i < months.length; i++){
            dataEntries.add(new ValueDataEntry(months[i], earnings[i]));
        }

        pie.data(dataEntries);
        anyChartView.setChart(pie);

    }
}

File: app/src/main/java/com/example/kalkulatorbmi/DeathActivity.kt
package com.example.kalkulatorbmi

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.anychart.AnyChart
import com.anychart.AnyChartView
import com.anychart.chart.common.dataentry.DataEntry
import com.anychart.chart.common.dataentry.ValueDataEntry
import java.util.*

class DeathActivity : AppCompatActivity() {
    var anyChartView: AnyChartView? = null
    var months = arrayOf("Jan", "Feb", "Mar")
    var earnings = intArrayOf(12000, 8009, 10000)
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_death)
        anyChartView = findViewById(R.id.any_chart_view)
        setupPieChart()
    }

    fun setupPieChart() {
        val pie = AnyChart.pie()
        val dataEntries: MutableList<DataEntry> = ArrayList()
        for (i in months.indices) {
            dataEntries.add(ValueDataEntry(months[i], earnings[i]))
        }
        pie.data(dataEntries)
        anyChartView!!.setChart(pie)
    }
}